.changeset/neat-snakes-bow.md

```diff
@@ -0,0 +1,7 @@
+---
+"@next-fetch/core-plugin": patch
+"@next-fetch/react-query": patch
+"@next-fetch/swr": patch
+---
+
+Restrict FormProps to explicitly prevent users from using unsupported encTypes.
```

Comment:
<picture data-single-emoji=":smart:" title=":smart:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..PsFm1h9z6yJnKuUg.5TYu3YVFR-SKJySJA295Hx0HQthgeKXIpIeMLfe1qZVdOFd8-r0dEeHOiWWJHRIUGQnuwGe0IvlC7y-HKiDmYJGiCa_QBOo6EMZiY_Szj3fvFbsjENw.xt9B2B-iqwZg5HVsC8jxWA" alt=":smart:" align="absmiddle"></picture>

we can support multipart once we have a good api for that <picture data-single-emoji=":just-right:" title=":just-right:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..NuLZIjn1mShvA654.wqQtYq7xMoW-LpGsxk-7p9UJvFK2STYX1eZ9-1PcNlY0Gv-MxYRI6qHsna2wOBvhJTW7J2aslOZ5MeAp9PzUxL1HGJNseFTQTgdvC_50AcjFVoYHrLcVoAEkmQ.-ipytRCX2o1sUwYZ2xRzbQ" alt=":just-right:" align="absmiddle"></picture>